projects/sidechain-connector-service-typescript/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { topology, grpc } from "orbs-common-library";
+import SidechainConnectorService, { SidechainConnectorServiceOptions } from "./service";
+
+const server = grpc.sidechainConnectorServer({
+  endpoint: topology.endpoint,
+  service: new SidechainConnectorService({
+    ethereumNodeHttpAddress: process.argv[3] // argv[2] is "taken" by orbs-common-library/topology.ts
```

Comment:
What about adding a more convenient ARGV parser? Personally, I like `yargs` (it also sounds like a pirate!).

Reply:
I'd rather have it in `topology.ts` so I can maybe pass it as an environment variable later.

Reply:
@netoneko I've also addresses this in the general comment below. We need to think of how are we going to deploy an Ethereum full node instance/pod and then bind and "discover" them alongside the Ethereum connector service (which will be a docker of its own, eventually). Let's discuss together with @idandash tomorrow.